fdm-app/app/routes/user.settings.api-keys.tsx

```
@@ -0,0 +1,453 @@
+            const result = await authClient.apiKey.list({
+                query: { sortBy: "createdAt", sortDirection: "desc" },
+            })
+            setKeys((result.data?.apiKeys as ApiKey[]) ?? [])
+        } catch {
+            toast.error("Kon API-sleutels niet laden.")
+        } finally {
+            setIsLoading(false)
+        }
+    }
+
+    useEffect(() => {
```

Biome (2.4.15)

[error] This hook does not specify its dependency on loadKeys.

(lint/correctness/useExhaustiveDependencies)